LICENSE

```diff
@@ -221,6 +221,7 @@ connector/spark-ganglia-lgpl/src/main/java/com/codahale/metrics/ganglia/GangliaR
 Python Software Foundation License
 ----------------------------------
 
+python/pyspark/loose_version.py
 python/docs/source/_static/copybutton.js
```

Comment:
> I believe it's compatible but we need to take a look once more before doing that. It's because our Apache Spark (up to 3.5.0) binary distribution doesn't include Python Software Foundation yet.

So actually we already have PSF License stuff?

Reply:
Yes and no~

Yes, we had `copybutton.js` file.
```
spark-3.5.0-bin-hadoop3:$ find . -name copybutton.js
./python/docs/source/_static/copybutton.js
```

But, still no. We didn't have no PSF entry in `LICENSE-binary` which is a part of Apache Spark binary distribution. So, I added in this PR.

Reply:
I'm not sure about `copybutton.js` but we need to add loose_version to binary because we had `python/pyspark/cloudpickle.py` and `python/pyspark/join.py` in BSD 3-Clause section. So, I added it to LICENSE-binary too.